compliance/http/src/test/java/org/eclipse/rdf4j/http/server/ProtocolTest.java

```diff
@@ -482,6 +516,62 @@ public void run() {
 		threadPool.shutdownNow();
 	}
 
+	private Set<Namespace> getNamespaces(String location)
+		throws Exception
+	{
+		URL url = new URL(location);
+		HttpURLConnection conn = (HttpURLConnection)url.openConnection();
+		try {
+			conn.setRequestProperty("Accept", "text/csv; charset=utf-8");
+			conn.connect();
+
+			InputStream connIs = conn.getInputStream();
+			try {
+				int responseCode = conn.getResponseCode();
+				if (responseCode != HttpURLConnection.HTTP_OK) {
+					String response = "location " + location + " responded: " + conn.getResponseMessage()
+							+ " (" + responseCode + ")";
+					fail(response);
+				}
+
+				String responseText = IOUtil.readString(connIs);
+				String[] rows = responseText.split("\\r?\\n");
```

Comment:
As we already have opencsv on the classpath at this point, inherited from queryresultio-text, it may be useful to rewrite the (very simple in this case) parsing code using opencsv, but that shouldn't stop this pull request from being accepted.